backend/calc.py

```diff
@@ -170,10 +173,21 @@ def crit_buffs(cls):
     def dh_buffs(cls):
         return {cls.BV, cls.BARD_DH, cls.DEVILMENT}
 
-    def avg_buff_effect(self):
+    @classmethod
+    def raid_buffs(cls):
+        return {cls.DIV, cls.TRICK, cls.BROTHERHOOD, cls.TECH, cls.DEVOTION, cls.EMBOLDEN}
+
+    def avg_buff_effect(self, job):
+        if self.name == 'EMBOLDEN':
+            if job == Jobs.RDM or job.role in {Roles.TANK, Roles.MELEE, Roles.RANGED}:
+                decay_interval = 4
+                decay_rate = 0.2
```

Comment:
I believe this should be 0.02 since Embolden decreases by 2% each time it ticks down (Though that is 20% of the whole).

Either 0.02 here or line 186 could be changed to accomodate this as a fraction of the total buff, rather than the actual decrease amount: `total += self.multiplier * (1 - decay_rate * i) * decay_interval / self.cd`